lib/install/validate/config_to_data.go

```diff
@@ -45,28 +45,30 @@ type Finder interface {
 
 // SetDataFromVM set value based on VCH VM properties
 func SetDataFromVM(ctx context.Context, finder Finder, vm *vm.VirtualMachine, d *data.Data) error {
+	op := trace.FromContext(ctx, "SetDataFromVM")
```

Comment:
This one seems like it could have been a signature change as it's confined to vic-machine.
Following the same pattern as above?
Not needed for this PR if it causes significant churn - these changes can be incremental.

Reply:
I applied the pattern from https://github.com/vmware/vic/pull/6777#discussion_r151894862 based on where the code was located, not how it was currently used.